apps/stops/lib/stop.ex

```diff
@@ -35,7 +35,7 @@ defmodule Stops.Stop do
           id: id_t,
           parent_id: id_t | nil,
           child_ids: [id_t],
-          name: String.t(),
+          name: String.t() | nil,
```

Comment:
Not sure if this change is needed, given we appear to be working with an empty string in this case.

Reply:
I can't tell if we are ever getting `nil` in this instance, but it seemed possible and even if not if you look up at the defstruct `name` defaults to `nil`, so this seems more correct.